asyncio_dgram/aio.py

```diff
@@ -213,15 +213,20 @@ def resume_writing(self):
         super().resume_writing()
 
 
-async def bind(addr):
+async def bind(addr, reuse_port=None):
```

Comment:
Please squash this commit (and the next one) into the previous commit.  I want to be sure that every commit in `master` will pass `make format lint test`.  This enables scripting git-bisect more easily.